test/iron-image.html

```diff
@@ -166,9 +166,14 @@
                 assert.equal(image.loading, false, 'ok image loading = false');
                 assert.equal(image.loaded, true, 'ok image loaded = true');
                 assert.equal(image.error, false, 'ok image error = false');
-                assert.equal(getComputedStyle(image.$.placeholder).display, 'none', 'placeholder has style.display = none');
 
-                done();
+                // TODO(bicknellr): Remove these `setTimeout`s or otherwise
```

Comment:
I need to take another look at these `setTimeout` calls and make sure they're still necessary. The original bug was deleted but I think it was Polymer/polymer#3910. (`loaded-changed` fires before the iron-image has had an opportunity to update styles on elements in its shadow root.)